tidb-cloud/data-service-manage-github-connection.md

```diff
@@ -0,0 +1,161 @@
+
+To import configurations of an existing Data App to a new Data App, take the following steps:
+
+1. Copy configuration files of the existing Data App to a new branch or directory on GitHub.
+2. On the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project, [create a new Data App](/tidb-cloud/data-service-manage-data-app.md#create-a-data-app) without connecting to GitHub.
+3. [Connect your new Data App to GitHub](#step-1-connect-your-data-app-to-github) with automatic deployment enabled. When you specify the target repository, branch, and directory for your new Data App, use your new path with the copied configuration files.
+4. Get the ID of your new Data App. You can click the name of your new Data App in the left pane and get the App ID in the **Data App Properties** area of the right pane.
+5. In your new path on GitHub, update the `app_id` in the `datapp_config.json` file to the ID you get, and then commit the changes.
+
+After the file changes are committed, TiDB Cloud will automatically deploy your new Data App with the latest changes. You can view the deployment status and commit information in the deployment history.
+
+## Edit GitHub connection
+
+If you want to edit the GitHub connection for your Data App (such as switching the repository, branch, and directory), perform the following steps.
+
+1. Navigate to the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project.
+2. In the left pane, click the name of your target Data App to view its details.
+3. In the **Connect to GitHub** area, click <svg width="16" height="16" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" color="gray.1"><path d="M11 3.99998H6.8C5.11984 3.99998 4.27976 3.99998 3.63803 4.32696C3.07354 4.61458 2.6146 5.07353 2.32698 5.63801C2 6.27975 2 7.11983 2 8.79998V17.2C2 18.8801 2 19.7202 2.32698 20.362C2.6146 20.9264 3.07354 21.3854 3.63803 21.673C4.27976 22 5.11984 22 6.8 22H15.2C16.8802 22 17.7202 22 18.362 21.673C18.9265 21.3854 19.3854 20.9264 19.673 20.362C20 19.7202 20 18.8801 20 17.2V13M7.99997 16H9.67452C10.1637 16 10.4083 16 10.6385 15.9447C10.8425 15.8957 11.0376 15.8149 11.2166 15.7053C11.4184 15.5816 11.5914 15.4086 11.9373 15.0627L21.5 5.49998C22.3284 4.67156 22.3284 3.32841 21.5 2.49998C20.6716 1.67156 19.3284 1.67155 18.5 2.49998L8.93723 12.0627C8.59133 12.4086 8.41838 12.5816 8.29469 12.7834C8.18504 12.9624 8.10423 13.1574 8.05523 13.3615C7.99997 13.5917 7.99997 13.8363 7.99997 14.3255V16Z" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></path></svg>. A dialog box for connection settings is displayed.
+4. In the dialog box, modify the repository, branch, and directory of your Data App.
+
+    > **Note:**
+    >
+    > - The directory must start with a slash. For example, `/mydata`. If the directory you specified does not exist in the target repository and branch, it will be created automatically.
```

Comment:
```suggestion
    > - The directory must start with a slash (`/`). For example, `/mydata`. If the directory you specified does not exist in the target repository and branch, it will be created automatically.
```